packages/core/src/payment/strategies/braintree/braintree-sdk-creator.ts

```diff
@@ -71,7 +72,7 @@ export default class BraintreeSDKCreator {
     }
 
     async getPaypalCheckout(
-        config: { currency: string },
+        config: { currency: string; indent?: BraintreeIntent },
```

Comment:
Why are we calling it `indent` here? Is that a typo?

Reply:
Yeah, its a typo :(